plugins/builds/getLatestBuild.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const buildSchema = schema.models.build.get;
+const jobIdSchema = joi.string().regex(/^[0-9]+$/);
+const statusSchema = joi.reach(schema.models.build.base, 'status');
+
+module.exports = () => ({
+    method: 'GET',
+    path: '/builds/latest',
+    config: {
+        description: 'Get latest build for job',
+        notes: 'Returns latest build for job, possibly restricted to builds with specific status',
+        tags: ['api', 'builds'],
+        auth: {
+            strategies: ['token'],
+            scope: ['user', 'build', 'pipeline']
+        },
+        plugins: {
+            'hapi-swagger': {
+                security: [{ token: [] }]
+            }
+        },
+        handler: (request, reply) => {
+            const { buildFactory } = request.server.app;
+            const { jobId, status } = request.query;
+
+            const listConfig = {
+                params: {
+                    jobId
+                }
+            };
+
+            if (status) {
+                listConfig.params.status = status;
+            }
+
+            return buildFactory
+                .list(listConfig)
```

Comment:
@InderH  Good, just not sure why `listConfig` is an object consists params? @klu909 is this the way the APIs written?

Reply:
config.params are the parameters that are used to filter when doing buildFactory.list

https://github.com/screwdriver-cd/models/blob/56e3ce9b02c68b99ca5848f3e9e029702f742f11/lib/buildFactory.js#L296